src/app.service.js

```diff
@@ -4,12 +4,16 @@ import jsdom from "jsdom";
 import { HttpError, HttpStatus } from "./utils/http";
 
 const EXCLUDED_TAGS_REGEX =
-  /^(button|img|nav|aside|footer|audio|canvas|embed|figure|iframe|map|area|noscript|object|option|optgroup|picture|progress|script|select|source|style|svg)$/i;
-const MAIN_CONTENT_TAGS_REGEX = /^(main|article|section|header)$/i;
+  /^(button|img|nav|aside|footer|audio|canvas|embed|iframe|map|area|noscript|object|option|optgroup|picture|progress|script|select|source|style|svg|meta)$/i;
+const MAIN_CONTENT_TAGS_REGEX = /^(main|article|section)$/i;
 const ENTER_REGEX = /[\n]/g;
 const TAB_REGEX = /[\t]/g;
 const WHITESPACE_REGEX = /\s+/g;
 const STIE_NAME_REGEX = /https?:\/\/(?:www\.)?([^\.\/]+)/;
```

Comment:
```suggestion
const SITE_NAME_REGEX = /https?:\/\/(?:www\.)?([^\.\/]+)/;
```
특정 사이트의 이름을 추출하기 위한 정규표현식으로 보이는데 스펠링 확인 부탁드립니다!